Example/Tests/Tests.m

```diff
@@ -934,8 +981,93 @@ - (ADBMediaObject *)createWithProperties:(NSDictionary *)properties andEventType
             });
 
         });
+    });
+});
+
+describe(@"SEGPlaybackDelegate", ^{
+    __block SEGPlaybackDelegate *playbackDelegate;
+    __block SEGPlaybackDelegate *mockDelegate;
+    __block SEGMockPlaybackDelegateFactory *mockPlaybackDelegateFactory = [[SEGMockPlaybackDelegateFactory alloc] init];
 
+    beforeEach(^{
+        mockPlaybackDelegateFactory.playbackDelegate = mockDelegate;
+        mockDelegate = mock([SEGPlaybackDelegate class]);
+
+        playbackDelegate = [[SEGPlaybackDelegate alloc] initWithDelegate:mockDelegate];
     });
 
+    describe(@"SEGPlaybackDelegate", ^{
+        it(@"sets initialTime on initialization", ^{
+            long currentTime = CFAbsoluteTimeGetCurrent();
+            assertThatLong(playbackDelegate.initialTime, equalToLong(currentTime));
+        });
+
+        it(@"gets incrementPlayheadPosition if not paused", ^{
+            playbackDelegate.isPaused = false;
+            [playbackDelegate getCurrentPlaybackTime];
+            [verify(mockDelegate) incrementPlayheadPosition];
```

Comment:
I'm not sure I understand this specific test. Why are you expecting an interaction with the mock here?

Additionally why are these tests using a mock att all? Seems like they can just test the real delegate.